src/mixins/experiment.js

```diff
@@ -11,12 +11,18 @@ export default function experimentsMixin(
         const { name, $activeVariants } = this.$exp;
         if (
           this.$exp.isEligible &&
-          !this.$exp.isEligible({ route: this.$route })
+          !this.$exp.isEligible({
+            route: {
+              name: this.getRouteBaseName(this.$route),
+              query: this.$route.query,
+              params: this.$route.query,
+            },
```

Comment:
```suggestion
            route: {
              name: this.getRouteBaseName(this.$route),
              query: this.$route.query,
              params: this.$route.params,
            },
```
I think this is more simple
```suggestion
            route: {
              ...this.$route,
              name: this.getRouteBaseName(this.$route),
            },
```